src/main.rs

```diff
@@ -1,4 +1,88 @@
-fn main() {
-    println!("Hello, world!");
+use axum::{
+    body::Body,
+    error_handling::HandleErrorLayer,
+    handler::Handler,
+    http::{uri, Request, StatusCode},
+    response::{IntoResponse, Response},
+    routing::get,
+    BoxError, Router, Server,
+};
+use std::convert::Infallible;
+use tower::{filter::AsyncFilterLayer, util::AndThenLayer, ServiceBuilder};
+
+fn init_router() -> Router {
+    let middleware_stack = ServiceBuilder::new()
+        .layer(HandleErrorLayer::new(|error| async move {
+            (
+                StatusCode::INTERNAL_SERVER_ERROR,
+                format!("Unhandled Internal Error: {}", error),
+            )
+        }))
+        .layer(AsyncFilterLayer::new(map_request))
+        .layer(AndThenLayer::new(map_response));
+
+    let meta_routes = Router::new()
+        .route("/", get(root_handler))
+        .route("/ping", get(heartbeat_handler));
+
+    let api_routes = Router::new()
+        .nest("/contacts", routes::contacts::router())
+        .nest("/dictionary", routes::dictionary::router())
+        .nest("/faqs", routes::faqs::router())
+        .nest("/food", routes::food::router())
+        .nest("/printing", routes::printing::router())
+        .nest("/spaces", routes::spaces::router())
+        .nest("/tools", routes::tools::router())
+        .nest("/transit", routes::transit::router())
+        .nest("/webcams", routes::webcams::router());
+
+    Router::new()
+        .nest("/", meta_routes)
+        .nest("/api", api_routes)
+        .layer(middleware_stack)
+        .fallback(fallback.into_service())
+}
+
+#[tokio::main]
+async fn main() {
+    tracing_subscriber::fmt::init();
+
+    let app = init_router();
+
+    Server::bind(&"0.0.0.0:3000".parse().unwrap())
+        .serve(app.into_make_service())
+        .await
+        .expect("server failed to exit successfully");
+}
+
+#[allow(clippy::unused_async)]
+async fn map_request(req: Request<Body>) -> Result<Request<Body>, BoxError> {
+    let (mut parts, body) = req.into_parts();
+
+    let new_path = parts.uri.path().replace("/v1", "");
```

Comment:
I believe we still want `map_request` and `map_response` around for when we interact with them shortly but I think the removal of "/v1" is not necessary any longer now that we are not versioning.